templates/web/pages/product/product_page.tt.html

```diff
@@ -10,7 +10,9 @@
     [% END %]
 [% END %]
 
-[% INCLUDE web/common/includes/off_webcomponents_configuration.tt.html language_code = lc country_code = cc robotoff_domain = robotoff_url images_domain = images_subdomain api_domain = formatted_subdomain %]
+[% INCLUDE web/common/includes/off_webcomponents_configuration.tt.html language_code = lc country_code = cc
+robotoff_domain = robotoff_url images_domain = images_subdomain api_domain = formatted_subdomain folksonomy_uri =
+"https://api.folksonomy.openfoodfacts.org" %]
```

Comment:
Of course, it will have to come from configuration, so you have to pass the url in the context at the point in the code where this template is called.